www/js/da.js

```diff
@@ -159,9 +159,15 @@ function formatResults(data) {
     return $.map(data, function(v,i) { return formatEntry(v.xml); }).join("");
 }
 
-function formatEntry(data) {
-    var template = doT.template("<h3>{{=it.term}}</h3><div>{{=it.def}}</div><i class='far fa-bookmark'></i>",
+function formatEntry(data) { 	
+    var template = doT.template("<h3>{{=it.term}}</h3><div>{{=it.def}}</div><i class='far fa-bookmark' id='bookmark' title=''></i>",
```

Comment:
quase perfeito, mas uma pagina pode mostrar várias definições, e não podemos ter ids repetidos.
Sugestão, usar o word id para os tornar diferentes:

      id='bookmark{{=it.word_id}}'